net-drv-ts/doc/dox_resources/tags.rst

```diff
@@ -9,3 +9,53 @@ TRC Tags List
 =============
 
 List of tags set on various testing configurations to distinguish them.
+
+.. list-table::
+  :header-rows: 1
+
+  *
+    - Name
+    - Description
+    - Example
+  *
+    - *DRIVER*
+    - Driver name. Typically it is the first choice if it is not known
+      that the driver supports the feature, but HW does not have it.
+    - i40e, ice, sfc
+  *
+    - linux-mm:*VALUE*
+    - Linux kernel major and minor versions combined in single integer
+      to classify testing results which depend on it. May be used for
+      comparison: greater, equal, less.
+    - linux-mm:510
+  *
+    - max_rx_queues:*VALUE*
+    - Number of receive queues.
+      May be used for comparison: greater, equal, less.
+    - max_rx_queues:8
+  *
+    - max_tx_queues:*VALUE*
+    - Number of transmit queues.
+      May be used for comparison: greater, equal, less.
+    - max_tx_queues:8
+  *
+    - pci-*VENDIR_ID*
+    - PCI device vendor ID to classify vendor-dependent testing results
+    - pci-1af4
+  *
+    - pci-*VENDIR_ID*-*DEVICE_ID*
```

Comment:
VENDIR_ID looks like a missprint here and further in the file

Reply:
Many thanks, fixed